resources/author-check.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import os
+import sys
+import tomllib
+
+
+def error(msg):
+    print('ERROR:', msg, file=sys.stderr)
+    sys.exit(1)
+
+
+def check_author(author_path):
+    print(f'Checking {author_path}...')
+
+    with open(author_path, 'rb') as f:
+        meta = tomllib.load(f)
+
+        if 'name' not in meta:
+            error(f'No `name` in `meta` section of {author_path}')
+
+        if 'description' not in meta:
+            error(f'No `description` in `meta` section of {author_path}')
+
+        if 'external' not in meta:
+            error(f'No `external` in `meta` section of {author_path}')
+
+        # Logo is optional, but if defined a corresponding image must exist
+        if 'logo' in meta:
+            logo_id = meta['logo']
+            logo_filename = f'{logo_id}.png'
+            logo_path = os.path.join('common/images/author-images', logo_filename)
+            if not os.path.isfile(logo_path):
```

Comment:
should we also allow image URLs here?

Reply:
I prefer to have the images in the folder, seems more organized. We could allow for it to link to CS images for example. But I'm not sure about linking to external images

Reply:
Hmm, maybe I can do something like, if you insert a valid url we assume that's a valid external image. If you just add the filename, we then check if `common/images/author-images/` has that file, would that work?
So, either:
`image="singlestore.png"`, checking if `common/images/author-images/singlestore.png` exists
or
`image=https://externalwebsite.com/image.png`
Would that work?

Reply:
Yeah sounds good